### PR TITLE
Update version and localization

### PR DESCRIPTION
**Pull request by ho-man-chan**

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -20,18 +20,18 @@ For installation instructions, visit [docs.pressbooks.org/installation](https://
...
 == Upgrade Notice ==
-= 5.15.0 =
+= 5.15.1 =
 
-* Pressbooks 5.15.0 requires PHP >= 7.1.
-* Pressbooks 5.15.0 requires [WordPress 5.4](https://wordpress.org/support/wordpress-version/version-5-3/)
-* Pressbooks 5.15.0 requires [McLuhan >= 2.10.2](https://github.com/pressbooks/pressbooks-book/)
-* Pressbooks 5.15.0 supports integration with [Sentry](https://sentry.io/)
+* Pressbooks 5.15.1 requires PHP >= 7.1.
+* Pressbooks 5.15.1 requires [WordPress 5.4](https://wordpress.org/support/wordpress-version/version-5-3/)
```
Comment (SteelWagstaff) [inline readme.txt:32]: Should be https://wordpress.org/support/wordpress-version/version-5-4/

File: readme.txt [hunk 2]
```diff
@@ -20,18 +20,18 @@ For installation instructions, visit [docs.pressbooks.org/installation](https://
...
+* Pressbooks 5.15.1 requires [McLuhan >= 2.10.2](https://github.com/pressbooks/pressbooks-book/)
+* Pressbooks 5.15.1 supports integration with [Sentry](https://sentry.io/)
```
Comment (SteelWagstaff) [inline readme.txt:34]: Didn't we add support for Sentry to 5.15.0? https://github.com/pressbooks/pressbooks/releases/tag/5.15.0

Comment (SteelWagstaff) [reply readme.txt:34]: Nevermind. I understand the purpose of the note. Ignore suggestion.

File: readme.txt (hunk 1 above)
Comment (ho-man-chan) [reply readme.txt]: its updated. thanks

**Pull request merged**